docs/guides/Guide-Intro.md

```diff
@@ -20,6 +20,20 @@ react-native run-android # or:
 react-native run-ios
 ```
 
+If you are using `create-react-native-app` instead of `react-native init`, then:
+```sh
+# Create a new React Native App
+create-react-native-app SimpleApp
+cd SimpleApp
+
+# Install the latest version of react-navigation from npm
+npm install --save react-navigation
+
+# Run the new app
+npm start
+
+# This will start a development server for you and print a QR code in your terminal.
+
```

Comment:
Make sure to close this code block with. Would also be helpful to give htem some more context around how to verify it's running - such as

"This will start a development server for you and print a QR code in your terminal. Scan the QR code using the Expo app on your iOS or Android device."